{
    "title" : "The given key was not present in the dictionary.",
    "uri" : "http://answers.coveo.com/questions/3472/the-given-key-was-not-present-in-the-dictionary.html",
    "printableUri" : "https://answers.coveo.com/questions/3472/the-given-key-was-not-present-in-the-dictionary.html",
    "clickUri" : "https://answers.coveo.com/questions/3472/the-given-key-was-not-present-in-the-dictionary.html",
    "uniqueId" : "42.4247$http://answers.coveo.com/questions/3472/the-given-key-was-not-present-in-the-dictionary.html",
    "excerpt" : "The given key was not present in the dictionary. ... My project has a custom search page that gets search results using the Sitecore search API. ... The out of the box Coveo search page works fine ...",
    "firstSentences" : null,
    "summary" : null,
    "flags" : "HasHtmlVersion",
    "hasHtmlVersion" : true,
    "hasMobileHtmlVersion" : false,
    "score" : 1030,
    "percentScore" : 72.21627,
    "rankingInfo" : null,
    "rating" : 3.0,
    "isTopResult" : false,
    "isRecommendation" : false,
    "titleHighlights" : [ ],
    "firstSentencesHighlights" : [ ],
    "excerptHighlights" : [ ],
    "printableUriHighlights" : [ ],
    "summaryHighlights" : [ ],
    "parentResult" : null,
    "childResults" : [ ],
    "totalNumberOfChildResults" : 0,
    "raw" : {
      "systitle" : "The given key was not present in the dictionary.",
      "sysauthor" : "jkearns",
      "sysurihash" : "2s2jPpUyu1IUQTBE",
      "urihash" : "2s2jPpUyu1IUQTBE",
      "numberofanswers" : 1,
      "sysuri" : "https://answers.coveo.com/questions/3472/the-given-key-was-not-present-in-the-dictionary.html",
      "sysprintableuri" : "https://answers.coveo.com/questions/3472/the-given-key-was-not-present-in-the-dictionary.html",
      "sitemappriority" : 0.1,
      "systransactionid" : 160348,
      "questionaskeddatestring" : "·\n            Jan 07, 2015 at 04:12 PM ·",
      "tags" : [ "coveoforsitecore", "sitecore", "api" ],
      "sysconcepts" : "coveo ; list of Sitecore SearchResultItems ; GetItem ; PredicateBuilder ; upgrade instructions ; config file ; uniqueid field ; Uri property ; article link ; index source ; coveoQueryFieldPipeline",
      "concepts" : "coveo ; list of Sitecore SearchResultItems ; GetItem ; PredicateBuilder ; upgrade instructions ; config file ; uniqueid field ; Uri property ; article link ; index source ; coveoQueryFieldPipeline",
      "printableuri" : "https://answers.coveo.com/questions/3472/the-given-key-was-not-present-in-the-dictionary.html",
      "documenttype" : "WebPage",
      "sysindexeddate" : 1517941887000,
      "permanentid" : "1644d0f689b539fc6174f2a4897e02f6d46cdca64423e15fda5fb82c30e2",
      "syslanguage" : [ "English" ],
      "transactionid" : 160348,
      "title" : "The given key was not present in the dictionary.",
      "date" : 1420675200000,
      "objecttype" : "Answers",
      "sitemapchangefrequency" : "Monthly",
      "answerauthor" : [ "Jean-François L'Heureux" ],
      "hasanswer" : "true",
      "audience" : [ "Administrator", "Developer" ],
      "commentauthor" : [ "Jean-François L'Heureux \u2666\u2666", "jkearns", "jkearns", "Jean-François L'Heureux \u2666\u2666" ],
      "hasacceptedanswer" : "true",
      "sourcetype" : "Sitemap",
      "sysconnectortype" : "SitemapCrawler",
      "rowid" : 5447246,
      "numberofcomments" : 4,
      "size" : 80385,
      "sysdocumenttype" : "WebPage",
      "questionvotes" : 0,
      "clickableuri" : "https://answers.coveo.com/questions/3472/the-given-key-was-not-present-in-the-dictionary.html",
      "syssource" : "AnswersCloud",
      "orderingid" : 212384744911577608,
      "acceptedanswers" : 1,
      "syssize" : 80385,
      "sysdate" : 1420675200000,
      "author" : "jkearns",
      "source" : "AnswersCloud",
      "questionbodyhtml" : "<div class=\"question-body\">\n                <p>My project has a custom search page that gets search results using the Sitecore search API. The out of the box Coveo search page works fine but the custom search page gives me the error \"The given key was not present in the dictionary.\"</p> \n<p>To get results I'm using the Sitecore PredicateBuilder class to get a list of Sitecore SearchResultItems. The error occurs when I try to use SearchResultItem.GetItem() to get a Sitecore item. I've had trouble in the past where the query is actually the problem, and the error only appears when I try to read the result. This doesn't seem to be the case, since I am able to use functions like .Count(), so I am getting a countable number of search result items that are causing errors when I try to get the Sitecore item from them. Also, I get the error even when the predicate builder is just the default true value, so the predicate doesn't seem to be the issue.</p> \n<p>I noticed this problem after updating my Coveo version. The code works on Coveo for Sitecore revision 742 (the October release) but breaks on revision 863 (December). I'm not sure which one is the anomaly, or how to fix it. Has anyone experienced this before?</p>\n                </div>",
      "sitemaplastmodified" : 1420675200000,
      "collection" : "default",
      "syssourcetype" : "Sitemap",
      "indexeddate" : 1517941887000,
      "questionnumber" : 3472,
      "connectortype" : "SitemapCrawler",
      "filetype" : "html",
      "sysclickableuri" : "https://answers.coveo.com/questions/3472/the-given-key-was-not-present-in-the-dictionary.html",
      "sysfiletype" : "html",
      "language" : [ "English" ],
      "sitemapparenturl" : "http://answers.coveo.com/questions/1.xml",
      "questionauthor" : "jkearns",
      "sitename" : "Answers",
      "sysrowid" : 5447246,
      "uri" : "https://answers.coveo.com/questions/3472/the-given-key-was-not-present-in-the-dictionary.html",
      "syscollection" : "default",
      "sitelanguage" : [ "English" ]
    },
    "Title" : "The given key was not present in the dictionary.",
    "Uri" : "http://answers.coveo.com/questions/3472/the-given-key-was-not-present-in-the-dictionary.html",
    "PrintableUri" : "https://answers.coveo.com/questions/3472/the-given-key-was-not-present-in-the-dictionary.html",
    "ClickUri" : "https://answers.coveo.com/questions/3472/the-given-key-was-not-present-in-the-dictionary.html",
    "UniqueId" : "42.4247$http://answers.coveo.com/questions/3472/the-given-key-was-not-present-in-the-dictionary.html",
    "Excerpt" : "The given key was not present in the dictionary. ... My project has a custom search page that gets search results using the Sitecore search API. ... The out of the box Coveo search page works fine ...",
    "FirstSentences" : null
  }